module1/src/com/epam/cycles/HomeTask_1_3_2.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.cycles;
+
+public class HomeTask_1_3_2 {
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 10;
+        int h = 1;
+        int y ;
+        for(int x = a; x <=  b; x += h) {
+            if (x > 2) {
+                y = x;
+            } else {
+                y = -1 * x;
+            }
+
+            System.out.println(y);
```

Comment:
You can here just SOUT(abbreviation for - System.out.printl(...)) the result
```suggestion
                System.out.println("y = " + -x + " (x = " + x + ")");
            } else {
                System.out.println("y = " + x + " (x = " + x + ")");
            }
```